docs/CHANGELOG.md

```diff
@@ -1241,7 +1241,7 @@
 
 #### Rift Fixes
 
-+ Fixed Rift NPC shops being treated as overworld ones for selling items to them. -
++ Fixed Rift NPC shops being treated as over world ones for selling items to them. -
```

Comment:
This feels wrong

Reply:
agreed

Reply:
Yeah, no, the original was correct.

https://minecraft.wiki/w/Overworld